src/main/java/step3/RacingRule.java

```diff
@@ -0,0 +1,34 @@
+package step3;
+
+import java.util.List;
+import java.util.Random;
+
+public class RacingRule implements Rule {
+    static final int CAN_GO = 4;
```

Comment:
상수이므로 private static final int 면 더 좋을 것 같습니다 😄 
```suggestion
    private static final int CAN_GO = 4;
```